docs/content/guides/deploy-apps/environments/overview/index.md

```diff
@@ -41,6 +41,10 @@ Supported identity providers:
 
 - [Azure AD workload identity](https://azure.github.io/azure-workload-identity/docs/introduction.html)
 
+### Simulated environments
+
+You can optionally designate an environment as "simulated". When enabled, a simulated environment will not deploy any output any resources or run any Recipes when an application is deployed. This is useful for dry runs or testing.
```

Comment:
Do you think we should link up the resource schema here?

Reply:
Nah we have a link to the schema at the bottom of the page. Eventually we'll have a how-to guide but don't have that written yet